metaflow/plugins/argo/argo_workflows.py

```diff
@@ -140,7 +140,9 @@ def _sanitize(name):
         return name.replace("_", "-")
 
     @classmethod
-    def trigger(cls, name, parameters={}):
+    def trigger(cls, name, parameters=None):
+        if parameters is None:
```

Comment:
Any specific reason for this change here and in L:383?

Reply:
Just removing mutable default arguments. [Doc](https://docs.python-guide.org/writing/gotchas/)